specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-05-01-preview/apimallpolicies.json

```diff
@@ -0,0 +1,85 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ApiManagementClient",
+    "description": "REST API for getting policy status in an Azure API Management deployment.",
+    "version": "2023-05-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/allPolicies": {
```

Comment:
>allPolicies [](http://example.com/codeflow?start=128&length=11)

is this naming signed off ? 

Do you not have individial gets for each policy ?

Reply:
yes I have gotten confirmation from PM and my Manager

Reply:
For individual policy - Not at the moment the intent of this api is just to return collection
we have another apis that would return individual policy at various levels 
example products/policy, api/policy, api/operation/policy etc

Reply:
"policies" alone would have suffized

Reply:
We already have that

Reply:
Hey but we want all policies as in tenant policy, productPolicy, ApiPolicy, OperationPolicy are all shown together here 

Also I was under the impression we need to have policies/{policyID}

and currently we also have 
/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/products/{productId}/policies/policy

/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}/policies/policy

/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/api/{pitId}/operations/{oId}policies/policy